Samsung/A32-SystemUI/res/values/dimens.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item type="dimen" name="config_screenBrightnessSettingDefaultFloat">0.5019608</item>
```

Comment:
none of the config_xxxx are in sysui

Reply:
so that means my whole system ui config is useless?

Reply:
no you can keep the one not starting with config_

Reply:
oh okay. btw i am not exprienced with overlays. but is there a list of configs or something where i could know what all the configs are? i followed the guide and thats how i made these 2 overlays.